Jenkinsfile

```diff
@@ -36,6 +36,16 @@ pipeline {
       }
     }
 
+    stage('Package') {
+      steps {
+        script {
+          version = sh(returnStdout: true, script: 'grep "const PLUGIN_VERSION = " Bootstrap.php | cut -d= -f2 | tr "," " "| tr ";" " " | tr "\'" " "').trim()
```

Comment:
Shouldn't this be the commit-id or the last version?

Reply:
@mridang We've been using the version number in our package releases. This avoids the need to locally rename the package before uploading.
Should I use the commit hash?